### PR TITLE
fix: disable send no account

### PR DESCRIPTION
**Pull request by obany**
# Description of change

When no account has been selected for an internal transfer the send is disabled.
Improves bech32 error reporting

## Links to any relevant issues

Fixes https://github.com/iotaledger/firefly/issues/715

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

## How the change has been tested

Tested on windows

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: packages/shared/lib/utils.ts [hunk 1]
```diff
@@ -155,7 +155,16 @@ export const isValidHttpsUrl = (url) => {
  * @returns True if it matches the bech32 format.
```
Comment (begonaalvarezd) [inline packages/shared/lib/utils.ts:155]: I just realized this `@returns` is wrong, maybe we can rephrase it to something like `@returns error message string if it doesnt match the bench32 format`?

File: packages/shared/routes/dashboard/wallet/views/Send.svelte [hunk 2]
```diff
@@ -259,7 +256,12 @@
     {#if !$isTransferring}
         <div class="flex flex-row justify-between px-2">
             <Button secondary classes="-mx-2 w-1/2" onClick={() => handleBackClick()}>{locale('actions.cancel')}</Button>
-            <Button classes="-mx-2 w-1/2" onClick={() => handleSendClick()}>{locale('actions.send')}</Button>
+            <Button
+                classes="-mx-2 w-1/2"
+                onClick={() => handleSendClick()}
+                disabled={selectedSendType === SEND_TYPE.INTERNAL && !to}>
```
Comment (begonaalvarezd) [inline packages/shared/routes/dashboard/wallet/views/Send.svelte:262]: what about using error handlers here instead of disabling the button?

Comment (obany) [reply packages/shared/routes/dashboard/wallet/views/Send.svelte]: Done

File: packages/shared/lib/utils.ts (hunk 1 above)
Comment (obany) [reply packages/shared/lib/utils.ts]: Updated

**Review by begonaalvarezd — APPROVED** [review]
